Dockerfile

```diff
@@ -98,6 +99,16 @@ RUN --mount=type=cache,target=/root/.ccache \
 RUN mkdir -p ../../dist/linux-amd64-rocm/lib/ollama && \
     (cd /opt/rocm/lib && tar cf - rocblas/library) | (cd ../../dist/linux-amd64-rocm/lib/ollama && tar xf - )
 
+FROM --platform=linux/arm64 cosdt/cann:${ASCEND_VERSION} AS ascend-build-arm64
+ARG CMAKE_VERSION
+COPY ./scripts/rh_linux_deps.sh /
+RUN CMAKE_VERSION=${CMAKE_VERSION} sh /rh_linux_deps.sh
+ENV PATH /opt/rh/gcc-toolset-10/root/usr/bin:$PATH
+COPY --from=llm-code / /go/src/github.com/ollama/ollama/
+WORKDIR /go/src/github.com/ollama/ollama/llm/generate
+ARG CGO_CFLAGS
+RUN OLLAMA_SKIP_STATIC_GENERATE=1 OLLAMA_SKIP_CPU_GENERATE=1 sh gen_linux.sh
+
```

Comment:
Hi, thanks for the contribution! We deprecated `cosdt/cann` image a few months ago, please use [ascendai/cann](https://hub.docker.com/r/ascendai/cann) instead.

Reply:
Alse, all images support `linux/amd64` and `linux/arm64`, you can add amd64 support if you want.